package.json

```diff
@@ -39,7 +39,7 @@
     "@types/sinon-chai": "^3.2.4",
     "@typescript-eslint/eslint-plugin": "^3.7.0",
     "@typescript-eslint/parser": "^3.7.0",
-    "admin-bro": "^3.2.3",
+    "admin-bro": "^3.4.0",
```

Comment:
I didn't know how to solve issues with duplicate types declarations of `admin-bro`. That prevents linking that adapter when admin-bro versions are mismatching - so I just bumped admin-bro version